### ISSUE TITLE
Features contains a language reference: undefined

### ISSUE THREAD

**Issue by tjperry07** [DavidAnson/markdownlint#392]
This is probably an edge case, but when trying to link out to this specific link, it returns an error message. 

```
Link https://docs.microsoft.com/en-us/office365/servicedescriptions/exchange-online-archiving-service-description/archive-features contains a language reference: undefined
```

Markdown Text

```
Microsoft accounts that have a large number of folders (>100) sometimes enter partial states more frequently. To improve an Exchange account's performance and have it synchronize faster, we recommend using [Exchange's Archive feature](https://docs.microsoft.com/en-us/office365/servicedescriptions/exchange-online-archiving-service-description/archive-features) on any of the folders that do not need to be synced with Nylas. The email administrator should have more details about how their version of Exchange handles archiving. 
```

Using the base styles.

**Comment by DavidAnson** [comment]
Your example does not trigger any warnings (see link below). Also, that error message does not look like one this tool produces. Where are you seeing that?

https://dlaa.me/markdownlint/#%25m%23%20Issue%20392%0A%0AMicrosoft%20accounts%20that%20have%20a%20large%20number%20of%20folders%20(%3E100)%20sometimes%20enter%20partial%20states%20more%20frequently.%20To%20improve%20an%20Exchange%20account's%20performance%20and%20have%20it%20synchronize%20faster%2C%20we%20recommend%20using%20%5BExchange's%20Archive%20feature%5D(https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Foffice365%2Fservicedescriptions%2Fexchange-online-archiving-service-description%2Farchive-features)%20on%20any%20of%20the%20folders%20that%20do%20not%20need%20to%20be%20synced%20with%20Nylas.%20The%20email%20administrator%20should%20have%20more%20details%20about%20how%20their%20version%20of%20Exchange%20handles%20archiving.%0A

**Comment by tjperry07** [comment]
Okay, this is embarrassing. This will 100% PEBKAC. So, sorry to open the issue. I am running two linters and opened an issue on the wrong one. 🤦🏾‍♀️

**Comment by DavidAnson** [comment]
I forgive you because you said "PEBKAC". :)
